src/dune_init.ml

```diff
@@ -20,7 +20,8 @@ module Kind = struct
   let commands =
     [ "exe", Executable
     ; "lib", Library
-    ; "test", Test ]
+    ; "test", Test
```

Comment:
Thanks for all these fixes to bring the style in line with the idioms of the
project! I'll try to remember them for future contributions :)

If these kinds of formatting changes are significant enough to warrant
correction, I wonder if it would be worth codifying them under
https://github.com/ocaml/dune/blob/master/CONTRIBUTING.md#coding-style so you
can point contributors there.

I'll open a separate PR trying to put the principles behind these changes into
words there. That will help reinforce them for my own sake as well.

Reply:
Soon we'll have ocamlformat forcing its style on us. So there's really no necessity.

Reply:
Ah, nice!